packages/agent/src/session/SessionManager.ts

```diff
@@ -130,12 +130,12 @@ export class SessionManager {
     // Create agent if config provided
     if (agentConfig) {
       try {
-        const agent = new ClaudeSDKAgent(agentConfig, this.controllerBridge)
+        const agent = new CodexSDKAgent(agentConfig, this.controllerBridge)
```

Comment:
maybe we nee some factory for creating the agent and the type need not be hardcoded. Let's define this interface too